receiptui/src/androidTest/java/com/hyperwallet/android/ui/receipt/ListPrepaidCardReceiptsTest.java

```diff
@@ -81,18 +73,22 @@ protected Intent getActivityIntent() {
                     return intent;
                 }
             };
+    private TimeZone mDefaultTimeZone;
 
     @Before
     public void setup() {
         mMockWebServer.mockResponse().withHttpResponseCode(HTTP_OK).withBody(sResourceManager
                 .getResourceContent("authentication_token_response.json")).mock();
 
+        mDefaultTimeZone = TimeZone.getDefault();
```

Comment:
@skoong @vinotb-altmtrk if this starts to become more common in other tests, a test rule should be used. I'm ok with the current implementation as it's only in the receipt scope but if that becomes necessary in other places, use a rule

Reply:
@fmattos-hw Sure flavio. Noted.